pr_agent/servers/gitlab_webhook.py

```diff
@@ -159,8 +159,8 @@ async def inner(data: dict):
             except Exception as e:
                 get_logger().error(f"Failed to validate secret {request_token}: {e}")
                 return JSONResponse(status_code=status.HTTP_401_UNAUTHORIZED, content=jsonable_encoder({"message": "unauthorized"}))
-        elif get_settings().get("GITLAB.SHARED_SECRET"):
-            secret = get_settings().get("GITLAB.SHARED_SECRET")
+        elif get_settings().get("GITLAB.SHARED_SECRET") or get_settings().get("GITLAB.WEBHOOK_SECRET"):
+            secret = get_settings().get("GITLAB.SHARED_SECRET") or get_settings().get("GITLAB.WEBHOOK_SECRET")
```

Comment:
remove this duplication.

configs shoud have a single definition

Reply:
The decision is yours, ofc. 
The idea was not to duplicate but rather to guarantee backwards compatibility.
As for the var name, I only "shared_secret" used in a very specific context on the bitbucket webhook handler (when using a secret provider). The goal was to make things consistent between providers. Let me know if this is final then.

Reply:
Definitely remove, and we can merge.

the name appears in other places

![image](https://github.com/user-attachments/assets/1f0f8547-aae5-4ac9-b357-03b90df93ce4)

Reply:
removed